### PR TITLE
Update camera api

### PR DESCRIPTION
**Pull request by jamie-houston**
Updates camera api to remove deprecated camera api to replace with camerax

### CODE REVIEW

**Comment by jamie-houston** [comment]
Found another crash.  Fixing.

File: platforms/android/barcodescanner/src/main/java/com/salesforce/barcodescannerplugin/BarcodeAnalyzer.kt [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package com.salesforce.barcodescannerplugin
```
Comment (bhariharan) [inline platforms/android/barcodescanner/src/main/java/com/salesforce/barcodescannerplugin/BarcodeAnalyzer.kt]: Copyright?

File: platforms/android/demoapp/google-services.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (bhariharan) [inline platforms/android/demoapp/google-services.json]: Do we need this file? It appears to be in the plugin already - wouldn't it automatically get pulled in?

File: platforms/android/barcodescanner/google-services.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (bhariharan) [inline platforms/android/barcodescanner/google-services.json]: Does this get bundled as part of the plugin?

File: platforms/android/demoapp/google-services.json (hunk 2 above)
Comment (jamie-houston) [reply platforms/android/demoapp/google-services.json]: The issue is the UTs fail without it.  I'll see if there's a workaround.  Ideally this isn't included and must be provided by the consumer.

Comment (jamie-houston) [reply platforms/android/demoapp/google-services.json]: @bhariharan I figured out how to put google-services.json into an encoded circleci variable, so it's gone now :)

File: platforms/android/barcodescanner/build.gradle [hunk 4]
```diff
@@ -26,24 +26,35 @@ android {
...
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.salesforce.nimbus:nimbus:0.0.3"
+    implementation "com.salesforce.nimbus:nimbus:0.0.7"
     implementation 'com.google.firebase:firebase-ml-vision:24.0.1'
     implementation 'com.google.firebase:firebase-ml-vision-object-detection-model:19.0.3'
-    implementation 'com.google.android.material:material:1.2.0-alpha04'
+    implementation 'com.google.android.material:material:1.2.0-alpha05'
+    implementation "androidx.camera:camera-camera2:${camerax_version}"
+    implementation "androidx.camera:camera-core:${camerax_version}"
+    implementation "androidx.camera:camera-view:1.0.0-alpha08"
```
[the remark this answers is not in the document]
Comment (jamie-houston) [reply platforms/android/barcodescanner/build.gradle]: camera-view is versioned differently.  Currently alpha08

**Comment by jamie-houston** [comment]
@bhariharan I pushed all the updates if you want to review again.  Thanks for the feedback.

**Comment by bhariharan** [comment]
This PR addresses [this issue](https://github.com/salesforce/nimbus-plugin-barcode-scanner/issues/10).

**Pull request merged**